ttkwidgets/notebook.py

```diff
@@ -0,0 +1,935 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Copyright 2018-2019 Juliette Monsel <j_4321 at protonmail dot com>
+Copyright 2019 Dogeek
+
+Adapted from PyTkEditor - Python IDE's Notebook widget by
+Juliette Monsel. Adapted by Dogeek <https://github.com/Dogeek>
+
+PyTkEditor is distributed with the GNU GPL license.
+
+Notebook with draggable / scrollable tabs
+"""
+try:
+    import tkinter as tk
+    from tkinter import ttk
+except ImportError:
+    import Tkinter as tk
+    import ttk
+from ttkwidgets.utilities import move_widget, parse_geometry, coordinates_in_box
+
+
+class Tab(ttk.Frame):
+    """Notebook tab."""
+    def __init__(self, master=None, tab_nb=0, **kwargs):
+        """
+        :param master: parent widget
+        :param tab_nb: tab index
+        :param **kwargs: keyword arguments for ttk::Frame widgets
```

Comment:
These parameters are later passed to a `ttk.Label` initializer, note a `ttk.Frame` initializer. Is the docstring correct?